Server/ConcordiaCurriculumManager/Controllers/DossierReviewController.cs

```diff
@@ -48,7 +51,8 @@ public async Task<ActionResult> SubmitDossierForReview([FromBody, Required] Doss
     [SwaggerResponse(StatusCodes.Status200OK, "Dossier successfully rejected")]
     public async Task<ActionResult> RejectDossier([FromRoute, Required] Guid dossierId)
     {
-        await _dossierReviewService.RejectDossier(dossierId);
+        User user = _userAuthenticationService.GetCurrentUser().Result;
```

Comment:
For future reference, since this is an asynchronous function, you can use `User user = await _userAuthenticationService.GetCurrentUser();`. Using .Result instead of await will return the same result, but it blocks the thread (unlike await which frees the thread for other tasks). In larger systems with lots of users, you can run into resource starvation issues using .Result, but we're only expecting a small number of concurrent users for our system, so it's fine in this case.